src/actors/symbolication.rs

```diff
@@ -1015,6 +1011,13 @@ impl SymbolicationActor {
         self,
         request: SymbolicateStacktraces,
     ) -> Result<CompletedSymbolicationResponse, SymbolicationError> {
+        #[cfg(test)]
+        {
+            // For test_pending_request we need to yield to the executor because otherwise
+            // the entire symbolication completes without ever yielding.  But we need to
+            // test that responses.
+            crate::utils::futures::delay(std::time::Duration::from_millis(0)).await;
+        }
```

Comment:
We should decided what we'd like to do.  The way the `create_symbolication_request()` currently works is it keeps the response for 90s anyway, even when it is completed and polled once.  So the test does not need to rely on a pending response once, it could check that it can request the completion more than once which would also mean the functionality is tested.